docs/documentation/serve.md

```diff
@@ -6,54 +6,62 @@
 `ng serve` builds the application and starts a web server
 
 ## Options
-`--watch` (`-w`) rebuild when files change
+`--aot` Build using Ahead of Time compilation.
```

Comment:
serve inherits options from build, can you list just the serve specific options and refer to the build command for the rest?